### PR TITLE
Branch 1

### PR DESCRIPTION
**Pull request by sammarth-kapse**
Merge Please.

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Author : Sammarth Kapse
```
Comment (dilipchauhan14) [inline test.py:1]: Good Work.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # test_repository
 To test the lambda function's operation.
+See if Lambda Works Properly.
```
Comment (dilipchauhan14) [inline README.md:3]: Nice Description-1.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,4 @@
...
+And payload is perated well.
```
Comment (dilipchauhan14) [inline README.md:4]: Nice Description-2.

File: test.py [hunk 4]
```diff
@@ -89,6 +93,3 @@ def get_merged_by(merged_by):
         return None
     else:
         return merged_by[LOGIN]
-
```
Comment (dilipchauhan14) [inline test.py:92]: No use-1.

File: test.py [hunk 5]
```diff
@@ -89,6 +93,3 @@ def get_merged_by(merged_by):
...
-    # Footer
```
Comment (dilipchauhan14) [inline test.py:93]: No Use-2.

File: test.py [hunk 6]
```diff
@@ -89,6 +93,3 @@ def get_merged_by(merged_by):
...
-
```
Comment (dilipchauhan14) [inline test.py:94]: No Use -3.

**Review by dilipchauhan14 — COMMENTED** [review]
No Use.

**Review by dilipchauhan14 — APPROVED** [review]
Approving...

**Review by sammarth-kapse — COMMENTED** [review]
ok!!!!

**Pull request merged**